### DISCUSSION TITLE
💁🏻Not possible to enable 2FA auth phone for Ukraine.🇺🇦

### DISCUSSION THREAD

**Discussion by ArtemFedorchuk** [community/community#37223] [Other Feature Feedback, Questions, & Ideas]
I tried to set up my 2FA through my Ukrainian🇺🇦 phone number but it is not possible. Could you add Ukraine, because it's very strange and very difficult to log in somehow different

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ArtemFedorchuk! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by ArtemFedorchuk** [reply]
> Thanks for posting in the GitHub Community, @ArtemFedorchuk! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

Thank you for your help. I'll look forward to solving this issue

**Comment by RONIE70** [comment]
Generar un código a través de una aplicación TOTP
Si decides configurar una autenticación de dos factores mediante una aplicación TOTP en tu smartphone, puedes generar un código de autenticación para GitHub en cualquier momento. En la mayoría de los casos, el lanzamiento de la aplicación generará un código nuevo. Deberías consultar la documentación de la aplicación para conocer las instrucciones específicas.

Si eliminas las aplicaciones móviles después de configurar la autenticación de dos factores, deberás proporcionar tu código de recuperación para obtener acceso a tu cuenta. Para más información, vea "[Recuperación de la cuenta en caso de perder las credenciales de autenticación en dos fases](https://docs.github.com/es/articles/recovering-your-account-if-you-lose-your-2fa-credentials)"

Recibir un mensaje de texto
Si configuras una autenticación de dos factores mediante mensajes de texto, GitHub te enviará un mensaje de texto con tu código de autenticación.

Verificar con GitHub Mobile
Si instalaste e iniciaste sesión en GitHub Mobile, podrías elegir autenticarte con GitHub Mobile para la autenticación bifactorial.

Inicia sesión en GitHub con tu buscador, utilizando tu usuario y contraseña.

Si agregaste una llave de seguridad a tu cuenta, primero se te pedirá insertar y utilizar una llave de seguridad. Para omitir el uso de una clave de seguridad, haga clic en Autenticar con GitHub Mobile .
Desafío de la autenticación en dos fases en GitHub con "Autenticación con GitHub Mobile" resaltado

GitHub te enviará una notificación push para verificar tu intento de inicio de sesión. Abrir la notificación push o abrir la app de GitHub Mobile mostrará un mensaje que te pide aprobar o rechazar este intento de inicio de sesión.

Nota: Este mensaje podría indicarle que escriba un número de dos dígitos que se muestra dentro del explorador en el que ha iniciado sesión.

El reto de autenticación bifactorial con GitHub Mobile que requiere una entrada de dos dígitos

Cuando apruebes el intento de inicio de sesión utilizando GitHub Mobile, tu buscador completará el inicio de sesión automáticamente.
Si rechazas el intento de inicio de sesión, se prevendrá la finalización de la autenticación. Para más información, vea "[Protección de la cuenta y los datos](https://docs.github.com/es/authentication/keeping-your-account-and-data-secure)".
Usar autenticación de dos factores con la línea de comando
Después de habilitar 2FA, ya no usarás la contraseña para acceder a GitHub en la línea de comandos. En vez de ello tendrás que usar el Administrador de credenciales de Git, un token de acceso personal o una clave SSH.

Autenticación en la línea de comandos mediante el Administrador de credenciales de Git
El [Administrador de credenciales de Git](https://github.com/GitCredentialManager/git-credential-manager/blob/main/README.md) es un asistente de credenciales de Git seguro que se ejecuta en Windows, macOS y Linux. Para obtener más información sobre los asistentes de credenciales de Git, consulta [Evitar repeticiones](https://git-scm.com/docs/gitcredentials#_avoiding_repetition) en el libro de Git de Pro.

Las instrucciones de configuración varían en función del sistema operativo del equipo. Para obtener más información, consulta [Descargar e instalar](https://github.com/GitCredentialManager/git-credential-manager/blob/main/README.md#download-and-install) en el repositorio GitCredentialManager/git-credential-manager.

Autenticar en la línea de comando mediante HTTPS
Después de haber habilitado 2FA, debes crear un token de acceso personal para usar una contraseña al autenticar a GitHub en la línea de comando mediante las URL HTTPS.

Cuando se te solicite el nombre de usuario y la contraseña en la línea de comando, usa tu nombre de usuario GitHub y el token de acceso personal. La indicación de la línea de comando no especificará que debes ingresar tu token de acceso personal cuando se te solicite la contraseña.

Para más información, vea "[Creación de un token de acceso personal](https://docs.github.com/es/github/authenticating-to-github/creating-a-personal-access-token)".

Autenticar en la línea de comandos mediante SSH
La habilitación de 2FA no cambia el modo de autenticar a GitHub en la línea de comando mediante las URL SSH. Para más información sobre cómo configurar y usar una clave SSH, vea "[Conexión a GitHub con SSH](https://docs.github.com/es/articles/connecting-to-github-with-ssh)".

Usar autenticación de dos factores para acceder a un repositorio mediante Subversion
Cuando accedas a un repositorio mediante Subversion, debes proporcionar un token de acceso personal en lugar de ingresar tu contraseña. Para más información, vea "[Creación de un token de acceso personal](https://docs.github.com/es/github/authenticating-to-github/creating-a-personal-access-token)".

Solución de problemas
Si pierdes el acceso a tus credenciales de autenticación de dos factores, puedes usar tus códigos de recuperación u otro método de recuperación (si has configurado uno) para recuperar el acceso a tu cuenta. Para más información, vea "[Recuperación de la cuenta si se pierden las credenciales de 2FA](https://docs.github.com/es/articles/recovering-your-account-if-you-lose-your-2fa-credentials)".

Si tu autenticación falla varias veces, es posible que desees sincronizar el reloj de tu teléfono con tu proveedor móvil. Frecuentemente, esto involucra la verificación de la opción "Establecer automáticamente" en el reloj de tu teléfono, en lugar de brindar tu propia zona horaria.
